SU-HSDO/suhumsci
# SHS-5558: Bug: Broken hover link style on vertical card titles on FranceStanford with half-height gray background

# READY FOR REVIEW

## Summary
Fix broken hover link style on vertical card titles on FranceStanford with half-height gray background

## Need Review By (Date)
04/05

## Urgency
medium

## Steps to Test
1. Go to the following pages:
   - https://francestanford.suhumsci.loc/project-area-study/physicsmathematicscomputer-science
   - https://francestanford.suhumsci.loc/funding/collaborative-research-projects
   - https://francestanford.suhumsci.loc/project-area-study/humanities-arts
2. Confirm the hover link style on vertical card titles has more than the 50% of gray background

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
